appmonitoring/scripts/validate-aks-charts.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+az account set --subscription 66010356-d8a5-42d3-8593-6aaa3aeb1c11
```

Comment:
As we are doing these validations, what is the guarantee we are looking at resources created by this particular run and not some left-overs from previous runs?

Reply:
as long as pipelines are not running concurrently, it will run properly as it deletes all resources from the previous runs

Reply:
We can't rely on that. We need to figure out how we handle this. Do we create separate namespaces? Do we want to detect that another pipeline is running and fail? We'll get some weird unreliable results if we just ignore that.

Reply:
I believe we can just make the pipeline not run multiple jobs at once and wait if another is already running